src/platform/packages/shared/kbn-scout/src/playwright/config/create_config.ts

```diff
@@ -104,7 +104,7 @@ export function createPlaywrightConfig(options: ScoutPlaywrightOptions): Playwri
       timeout: 10000,
     },
 
-    outputDir: './output/test-artifacts', // For other test artifacts (screenshots, videos, traces)
+    outputDir: './.scout/output/test-artifacts', // For other test artifacts (screenshots, videos, traces)
```

Comment:
Looks like everything is going into this `./scout/output` directory, which makes the `output` path redundant.

Reply:
fixed in c1498b12ccd8b8f5cb7cd60ab787f99b065b6cde